Automattic/wc-calypso-bridge
# Introduce the remaining tasks bubble nudge under My Home > Home admin menu item

### Changes proposed in this Pull Request:



This PR introduces the remaining tasks bubble with the My Home menu item. Here's how it looks in wp-admin and the Calypso contexts:

|wp-admin|Calypso|
|---|---|
|![](https://cdn-std.droplr.net/files/acc_1208648/d7oHZy)|![](https://cdn-std.droplr.net/files/acc_1208648/ihnJus)|

Closes #1224 






### How to test the changes in this Pull Request:




1. Visit an ecommerce or WX site with at least one remaining setup task.
2. Notice that the bubble indicating the remaining tasks is missing from the My Home menu item.
3. Apply this patch, and hard-refresh.
4. Notice that the bubble is there.



### Other information:




### FOR PR REVIEWER ONLY:
